pkg/controller/machine/create.go

```diff
@@ -249,6 +250,19 @@ func (c *creator) checkApiserverAddress() {
 	}
 }
 
+func (c *creator) getNodeLabels() string {
+	var sb strings.Builder
+	labels := c.machine.GetLabels()
+	for k, v := range labels {
+		b := new(bytes.Buffer)
+		fmt.Fprintf(b, "%s=%s,", k, v)
```

Comment:
This can be simplified since you never use `bytes.Buffer` except to get a string.

```suggestion
		labele := fmt.Sprintf("%s=%s,", k, v)
```

Reply:
good comment, will do